web-scraper/scraping_cart_test.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
i expected the web-scraping to be super long and complication but you did a really great job of keeping everything concise! i think this is perfect for what we need in our tool